docs/UserGuide.md

```diff
@@ -165,7 +166,7 @@ Examples:
 
 ### Editing an event : `editevent`
 
-Format: `editevent INDEX sp/SPORT t/Faculty 1 t/Faculty 2 d/LocalDateTime v/Venue [pa/PARTICIPANTS]…​`
+Format: `editevent INDEX sp/SPORT t/FACULTY 1 t/FACULTY 2 d/LOCALDATETIME v/VENUE [pa/PARTICIPANTS]…​`
```

Comment:
Could we indicate all fields except the INDEX as optional?

Reply:
Line 180: Can we edit this to 'Edits the sport of the first event currently displayed to be `Chess`.
Line 181: Similar to the above, but 'Edits the sport of the second event to be...'